### ISSUE TITLE
[Suggestion]: Items for phase modification when resummoning with the withered beacon

### ISSUE THREAD

**Issue by BluePineappleGuy** [nonamecrackers2/crackers-wither-storm-mod#2061]
### Guidelines

- [x] By checking this box, I am saying that I have read the guidelines and believe to be following them to the best of my ability

### Description of feature request

Certain items could be added to the WS resummoning recipe to modify the phase of the resummoned storm. 
Example:
- phase -1 per golden apple
- phase +1 per tainted flesh block

**Comment by GearsawStudios** [comment]
Lets [A Nasty Surprise] to be gotten without config edits, I approve.

**Comment by TaintedBlock** [comment]
I like this idea. Should probably have a config for this to limit how high you can go in phases and if you can even do this at all

**Comment by Cod1ngGamer** [comment]
Yeah, makes sense, but golden apple stew would probably be better for starting at a lower phase.

**Comment by TaintedBlock** [comment]
> Yeah, makes sense, but golden apple stew would probably be better for starting at a lower phase.

seems a bit too expensive

**Comment by nonamecrackers2** [comment]
This is actually a good idea, surprised this wasn't thought of before
